rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,172 @@
+### DCP PR:
+
+# RFC Process
+
+## Summary
+
+The Human Cell Atlas Data Coordination Platform (HCA DCP) evolved informal, consensus-driven processes to coordinate and document technical and governance decisions.
```

Comment:
I would remove "consensus-driven" and "egalitarian" from the former and latter sentences.  I think that's the ideal we're going for, but in the early days of the DCP it was really a small group of people making reasonable choices within boxes given their understanding of the DCP.

Reply:
WFM. It was a direct quote from the white paper.